### ISSUE TITLE
Android WebView blocked by header restrisctions

### ISSUE THREAD

**Issue by hdimitrov1** [twilio/twilio-video.js#1260]
- [ ] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [ ] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

Calling the 

`isSupportedByTwilio()`

on an android webview browser with the default user agent returns false

**Actual behavior:**

 it should return true. This is basically the same as android chrome, so if I spoof the header it works fine.

As a hint, the headers for the browser on my device are

Webview:
```
Mozilla/5.0 (Linux; Android 10; ONEPLUS A6003 Build/QKQ1.190716.003; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/86.0.4240.110 Mobile Safari/537.36

```

Chrome:
```

Mozilla/5.0 (Linux; Android 10; ONEPLUS A6003) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.110 Mobile Safari/537.36
```

TODO

**Software versions:**

 - [ ] Browser(s): Android Webview
 - [ ] Operating System: Android
 - [ ] twilio-video.js: 2.7.2
 - [ ] Third-party libraries (e.g., Angular, React, etc.): n/a

**Comment by manjeshbhargav** [comment]
Hi @hdimitrov1 ,

The `isSupported` flag denotes whether Twilio officially supports a browser by including it in our QA test matrix and providing support for issues experienced on that browser. If it returns `false` it does not mean that you cannot use the SDK on the browser. The SDK can be used on any WebRTC compatible browser. `false` just means that Twilio does not provide support for issues you may see in this browser, and you have to use the SDK in this browser at your own risk.

Please go [here](https://www.twilio.com/docs/video/javascript#supported-browsers) to see which browsers are officially supported by Twilio.

Thanks,

Manjesh
